### PR TITLE
Support date,time color and thing configuration

### PR DESCRIPTION
**Pull request by aounhaider1**
fixes https://github.com/eclipse/smarthome/issues/1576

### CODE REVIEW

**Comment by maggu2810** [comment]
This branch has conflicts that must be resolved

**Comment by aounhaider1** [comment]
Conflicts resolved.

File: extensions/ui/org.eclipse.smarthome.ui.paper/web-src/css/components.css [hunk 1]
```diff
@@ -755,6 +755,7 @@ md-input-container .md-errors-spacer {
 	width: 0%;
 }
 
+<<<<<<< HEAD
```
Comment (kaikreuzer) [inline extensions/ui/org.eclipse.smarthome.ui.paper/web-src/css/components.css]: this does not look like a successful resolution of merge conflicts :-/

**Comment by aounhaider1** [comment]
Updated!

**Comment by kaikreuzer** [comment]
Thanks!

**Pull request merged**